src/app/invest/invest.component.ts

```diff
@@ -12,10 +12,11 @@ declare var $: any;
 @Component({
     selector: 'app-invest',
     templateUrl: './invest.component.html',
-    styleUrls: ['./invest.component.css'],
+    styleUrls: ['./invest.component.scss'],
     encapsulation: ViewEncapsulation.None
 })
 export class InvestComponent implements OnInit {
+    isOverview = false;
```

Comment:
why should invest component (which only renders inside secure layout) have `isOverview` property?